### PR TITLE
[Admin] Fix 500 error when showing cart in admin panel

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.11           |
| Bug fix?        | yes                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no |
| Related tickets | down ported version of https://github.com/Sylius/Sylius/pull/13831, fixes https://github.com/Sylius/Sylius/issues/13826 |
| License         | MIT                                                          |

### CODE REVIEW

**Review by Zales0123 — COMMENTED** [review]
It would be good to have some tests for that 🖖

**Comment by Zales0123** [comment]
Any updates, Łukasz? :)

File: src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php [hunk 1]
```diff
@@ -67,6 +69,14 @@ public function iSeeTheOrder(OrderInterface $order)
         $this->showPage->open(['id' => $order->getId()]);
     }
 
+    /**
+     * @When /^I try to view the summary of the (customer's latest cart)$/
+     */
+    public function iTryToViewTheSummaryOfTheCustomersLatestCart(?OrderInterface $cart): void
```
Comment (NoResponseMate) [inline src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php]:
```suggestion
    public function iTryToViewTheSummaryOfTheCustomersLatestCart(OrderInterface $cart): void
```

File: src/Sylius/Behat/Context/Transform/CartContext.php [hunk 2]
```diff
@@ -45,4 +50,20 @@ public function providePreviousCartToken(): ?string
 
         return $this->provideCartToken();
     }
+
+    /**
+     * @Transform /^(customer's latest cart)$/
+     */
+    public function provideLatestCart(): ?OrderInterface
```
Comment (NoResponseMate) [inline src/Sylius/Behat/Context/Transform/CartContext.php]:
```suggestion
    public function provideLatestCart(): OrderInterface
```

**Review by NoResponseMate — APPROVED** [review]

**Review by TheMilek — APPROVED** [review]

**Review by jakubtobiasz — APPROVED** [review]

**Comment by Rafikooo** [comment]
Thank you, @lchrusciel!

**Pull request merged**